src/mon/Monitor.cc

```diff
@@ -2561,8 +2563,8 @@ health_status_t Monitor::get_health_status(
 
   string summary;
   string *psummary = f ? nullptr : &summary;
-  for (auto& svc : paxos_service) {
-    r = std::min(r, svc->get_health_checks().dump_summary(
+  for (auto &s : paxos_service) {
```

Comment:
i'd suggest avoid these naming changes. also IMHO, `&` is part of the type, hence would be better to put it right next to `auto`. so `auto& s` or `auto& svc` is better.